.github/workflows/test.yml

```diff
@@ -0,0 +1,13 @@
+name: Test
+on: pull_request
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+  steps:
+  - uses: actions/checkout@v2
+  - uses: actions/setup-node@v2
+    with:
+      node-version: '14'
+      cache: 'npm'
+  - run: npm install
```

Comment:
You could cache the npm install like Cypress does here with this action

https://github.com/jane/cypress/blob/main/.github/workflows/check.yml#L18

Reply:
I wasn't certain what this would cache. Reading the docs again maybe its only caching the deps for this build so it could be tested against a matrix of node versions?

Reply:
Here is see more detail about how/why it works from his blog if it's of any help;

https://glebbahmutov.com/blog/do-not-let-npm-cache-snowball/

Reply:
Thanks. I remember reading this now. I'll update and avoid the potential snowball. ;)